storybook/stubs/shared/stores/send/TransactionStore.qml

```diff
@@ -170,10 +170,11 @@ QtObject {
         }
     }
 
-    function switchSenderAccountByAddress(address) {
-        for (let i = 0; i < senderAccounts.count; i++) {
-            if (senderAccounts.get(i).address === address) {
-                selectedSenderAccount = senderAccounts.get(i)
+    function setSenderAccountByAddress(address) {
+        for (let i = 0; i < accounts.count; i++) {
+            const acc = accounts.get(i)
+            if (acc.address === address && accounts.canSend) {
+                selectedSenderAccount = accounts.get(i)
```

Comment:
Not part of this PR, because it cannot be done differently easily, but the goal is to not keep any logic directly in stubs. Mocks should be created separately (and kept in e.g. `mocks` dir), or directly per page. Now it's not possible because of singleton stores - because of that singleton stores and substores in singeton store cannot be passed as an explicit dependency to ui components.